pandas/util/_decorators.py

```diff
@@ -177,6 +177,8 @@ def deprecate_kwarg(
     def _deprecate_kwarg(func: F) -> F:
         @wraps(func)
         def wrapper(*args, **kwargs) -> Callable[..., Any]:
+            __tracebackhide__ = True
```

Comment:
this is unrelated to the rest of the PR, just to de-noise tracebacks im seeing in other branches